app/models/git_hub_user.rb

```diff
@@ -15,8 +15,12 @@ def login
   # Public
   #
   def admin_organization_memberships
+    no_cache_headers = { 'Cache-Control' => 'no-cache, no-store' }
```

Comment:
Isn't that a server cache header? I assumed the problem was with the Octokit caching going on over in https://github.com/education/classroom/blob/master/config/initializers/octokit.rb#L2